### ISSUE TITLE
why do you usr cross entropy loss?

### ISSUE THREAD

**Issue by EthanGuan** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#17]
I noticed in the paper and original repo, L2 loss function is used.
In your implementation:

    # define loss function (criterion) and pptimizer
    criterion = nn.CrossEntropyLoss().cuda()

cross entropy loss is used.

does it really work?

**Comment by tensorboy** [comment]
I didn't successfully implement the training part, but the model is converted right. ;-D.

Best,
Wangpeng

**Comment by tensorboy** [comment]
Try `train.py`
